### DISCUSSION TITLE
Website failing to deploy

### DISCUSSION THREAD

**Discussion by strawbs1989** [community/community#119931] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi GitHub I believe there's a bug with your deployment 
Everytime I deploy to github I'm getting N error

Deploy static content to pages workflow run

Deploy static content to pages: all jobs have failed

Could you tell me why please???
![Screenshot_20240417-123136](https://github.com/community/community/assets/103792790/41980c52-af4e-4254-acdd-54be944a30c4)

**Comment by ebndev** [comment]
Welcome to the GitHub Community, @strawbs1989, we're happy you're here! 

I've gone ahead and moved this questions to our 🗒️ Pages category for you, which may be more applicable for this topic. You may also be more likely to get a response if you include the error messages / log that explains why your build may be failing.

Best of luck!

**Reply by strawbs1989** [reply, marked as answer]
Thank you but I've solved the GitHub issue now, it's just the last error uploading to cloudflare I've made a support there end no one seems to answer. Don't know if anyone can solve it here. If not I'll be moving domain over from them to here

**Reply by ebndev** [reply]
I removed your photo to not expose you API key but I did see that the error noted an internal issue on their end.

If they get back to you let us know if it gets resolved!

**Comment by strawbs1989** [comment]
I've posted on there community as well but not getting back to me, all I
want is for my site to start updating things again. It used to work months
ago but now this?

Its getting ridiculous!!! Cloudflare support? What support!! Lol

**Comment by strawbs1989** [comment]
I've just looked on there stats page yea it's there internal server, they
have alot of Ddos attacks to fix!
